### ISSUE TITLE
Wrong column name in 'person' result

### ISSUE THREAD

**Issue by mzico** [GluuFederation/oxTrust#794]
It should be 'Email_Address' / 'Email'; instead showing 'Status' again. 

How to reproduce: 

- 'Manage People'
- Search for any user

![screenshot_23](https://user-images.githubusercontent.com/7156097/34127502-08fbd654-e403-11e7-803e-4e0b69fedc47.png)

**Comment by mzico** [comment]
identity version: 

```
[root@ ~]# cat /opt/jetty-9.3/temp/jetty-localhost-8082-identity.war-_identity-any-8560950003527137009.dir/webapp/META-INF/MANIFEST.MF
Manifest-Version: 1.0
Implementation-Title: oxTrust Server
Implementation-Version: 3.1.1.Final
Archiver-Version: Plexus Archiver
Built-By: tomcat
Implementation-Build: cadf68802ca35923b592a52e052227c7ec169369
Implementation-Vendor-Id: org.xdi
Created-By: Apache Maven 3.3.9
Build-Jdk: 1.8.0_121

[root@sso ~]#
```

**Comment by shekhar16** [comment]
It is already fixed in 3.1.2.
![image](https://user-images.githubusercontent.com/15937500/34213166-6fd94236-e5c4-11e7-9e06-a30be5f3e74f.png)

**Comment by shekhar16** [comment]
https://github.com/GluuFederation/oxTrust/issues/756
